newswires/app/controllers/QueryController.scala

```diff
@@ -48,9 +63,10 @@ class QueryController(
     val maybePreset =
       request.getQueryString("preset").flatMap(SearchPresets.get)
 
-    val suppliersToExcludeByDefault =
-      if (featureSwitchProvider.ShowGuSuppliers.isOn()) Nil
-      else List("GuReuters", "GuAP")
+    val suppliersToExcludeByDefault: List[String] = List(
+      "UNAUTHED_EMAIL_FEED"
+    ) ++ (if (featureSwitchProvider.ShowGuSuppliers.isOn()) List.empty
+          else List("GuReuters", "GuAP"))
 
```

Comment:
I'd perhaps express this as: 

```
val suppliersToExcludeByDefault =
      if (featureSwitchProvider.ShowGuSuppliers.isOn()) List("UNAUTHED_EMAIL_FEED")
      else List("GuReuters", "GuAP", "UNAUTHED_EMAIL_FEED")
``` 

Even though it duplicates the email supplier, it feels a bit easier not to have to do string concat mentally. 


This this could also be good to perhaps move out of the main function and write a unit test for (more for documenting business logic than anything else)

Reply:
It would probably also be good to unit test the resolution of `suppliersExcl`. 

This expression: 

```
request.queryString
        .get("supplierExcl")
        .map(_.toList)
        .getOrElse(Nil) ++ suppliersToExcludeByDefault.filterNot(
        suppliers.contains
      )
```

Because there are a few variations on this depending on whether the switch is on, or what is present in the suppliers query. 

(Though the 
```
request.queryString
        .get("supplierExcl")
        .map(_.toList) 
```
could be computed else where so you don't have to worry about having a request object in the unit test. 

Let me know if it doesn't make sense